finals-2024-01-16/code/5263-angela-kostadinova/src/app/band-list/band-list.component.html

```diff
@@ -1 +1,78 @@
-<p>And the band list and everything else comes here</p>
+<p><div *ngFor="Bands">
```

Comment:
There is a big disconnect between the markup and the code, like they were independently done :)
+3 points for the markup